spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -85,7 +85,8 @@ const ConfigSteps = () => {
 		const handler = async (event: any) => {
 			if (event.origin !== originalUrl) return;
 			if (event.data?.code) {
-				await OAuthManagerInstance.finishOAuthFlow(event.data?.code, event.data?.state);
+				const success = await OAuthManagerInstance.finishOAuthFlow(event.data?.code, event.data?.state);
+				if (!success) return;
```

Comment:
@krazziekay maybe another PR, again. I don't know where to show the error message (like which UI component). 
For now, I just `NOT` call the following steps (setIsLoggin(true)) if the oauth fail.